Cargo.lock

```diff
@@ -2118,10 +2118,10 @@ dependencies = [
 
 [[package]]
 name = "sys-info"
-version = "0.5.1"
+version = "0.5.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "gcc 0.3.54 (registry+https://github.com/rust-lang/crates.io-index)",
+ "cc 1.0.28 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
This updated dependency should be the reason 😉

Reply:
Probably!